blog/_posts/2017-07-25-team-berlindiamonds-part-one.md

```diff
@@ -0,0 +1,42 @@
+As we already knew that our task would be to create a plugin for Discourse that allows the Admin User to backup his app to a clowd provider, we followed some tutorials on Discourse Meta on _How to install a plugin_ and looked at the [Backup-To-Dropbox-Plugin](https://github.com/xfalcox/discourse-backups-to-dropbox) in order to understand the structure of a Discourse plugin. So when we finally had our call with our Mentor the third day, he was gladly surprised how far we already were and he gave us our first really good feedback. We were very flattered and pleased with ourselves and so the following weeks went very smoothly with a lot of motivation from our side and allways encouraging feedback and really good explainations from our mentor's and our coaches side. Coding for the RailsGirls Summer of Code felt like being freshly in love and it made our cheeks blush from euphoria. Not kidding. 
+
+Our Coaches
+
+We are very lucky with our team of coaches. First of all we have a team of 2-3 coaches at our coaching company who are sitting next to us daily. They are very much in love with Ruby and asking them a question about an issue with the code mostly is followed by a little spontanious lesson enthusiastically explained with haptic examples like Monkey classes and banana methods. All of them have their fields of expertise and together they form a big universe of knowledge. We also have some geniuses in our remote coaches, who regularly fall into deep discussions about wether a plugin should be constructed like a gem or not and other fascinating questions in our slack channel. We follow these "fights of the giants" with big interest because there is a lot to learn from it.
+
+Our Plugin
+
+We already got quite far within the first weeks and finished building a plugin for Discourse that can upload backup files to Google Drive. If you want to check out the code and see our progress, please take a look at our [Github Repo](https://github.com/berlindiamonds/discourse-googledrive-backup). Although the plugin itself is technically working, as good coders we will still refactor our code and also create a reusable structure in order to build more backup plugins after that. 
+All in all this project has really made us feel more competent and growing with every problem we solve. We feel very empowered and happy and are very thankful for the chance of being part of the RailsGirls Summer of Code. Thanks to all contributers and organizers! We love it <3 
+![image](../img/blog/2017/2017-07-empowered-diamonds.jpg) Photo by: Markus from Absolventa
```

Comment:
we've got a special `<div class="image-credits">` for all image credits, could you add this to the post for the two images? you can see what it looks like in other teams' blog posts :) thanks!